### PR TITLE
Drop LVM, use custom growpart wrapper

### PR DESCRIPTION
**Pull request by cgwalters**
Closes: https://github.com/openshift/os/issues/293

### CODE REVIEW

**Comment by cgwalters** [comment]
Requires: https://github.com/coreos/coreos-assembler/pull/95

**Comment by cgwalters** [comment]
Actually unless we backport https://github.com/coreos/fedora-coreos-config/commit/0fdb511dbcb35b0472d909959b6346c915ae517c the commits interlock - pushing one without the other will be broken.

File: compose-post.sh [hunk 1]
```diff
@@ -9,6 +9,38 @@ ln -sfr /usr/lib/systemd/system/{multi-user,default}.target
...
+touch /var/lib/coreos-growpart.stamp
+EOF
+chmod a+x /usr/libexec/coreos-growpart
+cat > /usr/lib/systemd/system/coreos-growpart.service <<'EOF'
+[Unit]
+ConditionPathExists=!/var/lib/coreos-growpart.stamp
+[Service]
+ExecStart=/usr/libexec/coreos-growpart /
+RemainAfterExit=yes
+Before=sshd.service
```
Comment (dm0-) [inline compose-post.sh]: Isn't `Before=` required to be under the `[Unit]` section?

Comment (cgwalters) [reply compose-post.sh]: Apparently it's just ignored.

Comment (dm0-) [reply compose-post.sh]: Okay, I've submitted coreos/fedora-coreos-config#10.

Comment (cgwalters) [reply compose-post.sh]: https://github.com/coreos/fedora-coreos-config/pull/11

And updated below :arrow_down:

**Comment by cgwalters** [comment]
> @cgwalters Can that be backported in this PR set or do we need to keep them separate?

Wouldn't hurt but wouldn't help either - the problem is once we build a new `coreos-assembler:alpha`.  But we can just wait to do that until this merges.

**Pull request merged**